python/taichi/lang/ops.py

```diff
@@ -732,10 +732,18 @@ def expr_python_mod(a, b):
 def pow(x, a):  # pylint: disable=W0622
```

Comment:
Since we are here, would you mind updating this to `base` and `exp` so that they're more self-explanatory? :P

Reply:
Updated.